src/FondOfSpryker/Shared/Discount/Transfer/discount_custom_message.transfer.xml

```diff
@@ -15,4 +15,8 @@
     <transfer name="DiscountConfigurator">
         <property name="discountCustomMessages" type="DiscountCustomMessage[]"/>
     </transfer>
+
+    <transfer name="Quote">
+        <property name="usedNotAppliedDiscount" type="Discount[]"/>
```

Comment:
<property name="usedNotAppliedDiscounts" singular="usedNotAppliedDiscount" type="Discount[]"/> --> Would be better